style.css

```diff
@@ -26,29 +39,147 @@
 .pagination {
     display: flex;
     justify-content: center;
+
 }
 
 .card-body{
     color: black;
+
 }
 
+.card-header-tabs{
+  margin: 10px;
+
+} 
+
 .nav-link{
     border-radius: 20px;
+
 }
+
 .selected{
     background: yellow !important;
+
+}
+
+.cover h5, .cover p{
+  width: 80%;
+
+}
+
+/* SEARCH */
+
+.search-form{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  border-radius: 6px;
+
+}
+
+
+form.search-form input[type=text] {
+  padding: 10px;
+  font-size: 17px;
+  border: 1px solid grey;
+  float: left;
+  width: 70%;
+  background: #f1f1f1;
+  border-radius: 6px;
+
+}
+
+
+form.search-form button {
+  float: left;
+  width: 50px;
+  padding: 10px;
+  margin: 5px;
+  background: #2196F3;
+  color: white;
+  font-size: 17px;
+  border: 1px solid grey;
+  border-radius: 6px;
+  border-left: none;
+  cursor: pointer;
+
+}
+
+
+form.search-form button:hover {
+  background: #0b7dda;
+
+}
+
+form.search-form::after {
+  content: "";
+  clear: both;
+  display: table;
+
+}
+
+
+/* LOADER */
+
+.lds-facebook {
+  display: none;
+  position: relative;
+  top: 10%;
+  left: 45%;
+  width: 800px;
+  height: 800px;
+
+}
```

Comment:
The loder is not centered, i think it's better to use the absolute or fixed positions

Reply:
Loader stands in the middle horizantally. I doesn't  plant to centered it verticaly. Give me a sign, if i need to change something.